--- origsrc/cdk-5.0-20081105/Makefile.in	2008-10-30 18:40:47.000000000 -0500
+++ src/cdk-5.0-20081105/Makefile.in	2008-11-12 00:38:00.711699900 -0600
@@ -267,7 +267,7 @@
 all cdklib :: $(DEFAULT)
 
 $(CDKLIB) :: $(OBJECTS)
-	@echo creating $@; $(LIBTOOL_CREATE) $@ $(LIB_OBJECT)
+	@echo creating $@; $(LIBTOOL_CREATE) $@ $(LIB_OBJECT) $(LDFLAGS) -no-undefined $(LIBS)
 	@$(RANLIB) $@
 
 cdkshlib $(CDKSHLIB) :: $(CDKSHOBJS)
