--- origsrc/cdk-5.0-20120323/aclocal.m4	2012-03-19 05:47:09.000000000 -0500
+++ src/cdk-5.0-20120323/aclocal.m4	2012-03-25 18:49:41.039594500 -0500
@@ -38,10 +38,10 @@ dnl $3 = code to use if AC_ACVERSION is
 define(CF_ACVERSION_CHECK,
 [
 ifdef([m4_version_compare],
-[m4_if(m4_version_compare(m4_defn([AC_ACVERSION]), [$1]), -1, [$3], [$2])],
+[m4_if(m4_version_compare(m4_defn([m4_PACKAGE_VERSION]), [$1]), -1, [$3], [$2])],
 [CF_ACVERSION_COMPARE(
 AC_PREREQ_CANON(AC_PREREQ_SPLIT([$1])),
-AC_PREREQ_CANON(AC_PREREQ_SPLIT(AC_ACVERSION)), AC_ACVERSION, [$2], [$3])])])dnl
+AC_PREREQ_CANON(AC_PREREQ_SPLIT(m4_PACKAGE_VERSION)), m4_PACKAGE_VERSION, [$2], [$3])])])dnl
 dnl ---------------------------------------------------------------------------
 dnl CF_ACVERSION_COMPARE version: 2 updated: 2011/04/14 20:56:50
 dnl --------------------
@@ -60,7 +60,7 @@ dnl version of autoconf, or a patched ve
 dnl the patch-version).
 define(CF_AC_PREREQ,
 [CF_PREREQ_COMPARE(
-AC_PREREQ_CANON(AC_PREREQ_SPLIT(AC_ACVERSION)),
+AC_PREREQ_CANON(AC_PREREQ_SPLIT(m4_PACKAGE_VERSION)),
 AC_PREREQ_CANON(AC_PREREQ_SPLIT([$1])), [$1], [$2], [$3])])dnl
 dnl ---------------------------------------------------------------------------
 dnl CF_ADD_CFLAGS version: 10 updated: 2010/05/26 05:38:42
