--- origsrc/cdk-5.0-20060507/Makefile.in	2006-04-17 18:13:49.000000000 -0500
+++ src/cdk-5.0-20060507/Makefile.in	2008-04-13 23:58:14.453125000 -0500
@@ -265,7 +265,7 @@
 all cdklib :: $(DEFAULT)
 
 $(CDKLIB) :: $(OBJECTS)
-	@echo creating $@; $(LIBTOOL_CREATE) $@ $(LIB_OBJECT)
+	@echo creating $@; $(LIBTOOL_CREATE) $@ $(LIB_OBJECT) -no-undefined $(LIBS)
 	@$(RANLIB) $@
 
 cdkshlib $(CDKSHLIB) :: $(CDKSHOBJS)
--- origsrc/cdk-5.0-20060507/configure.in	2005-12-28 16:41:13.000000000 -0600
+++ src/cdk-5.0-20060507/configure.in	2008-04-14 00:01:55.421875000 -0500
@@ -2,7 +2,6 @@
 dnl Process this file with autoconf to produce a configure script for CDK.
 dnl
 AC_INIT(include/cdk.h)
-AC_CONFIG_HEADER(include/cdk_config.h:include/config.hin)
 
 CF_CHECK_CACHE
 CF_VERSION_INFO(cdk)
@@ -144,6 +143,7 @@
 dnl
 AC_OUTPUT(
 	include/cdk_version.h:include/cdk_version.hin
+	include/cdk_config.h:include/config.hin
 	Makefile
 	cli/Makefile
 	examples/Makefile
--- origsrc/cdk-5.0-20060507/include/config.hin	2000-01-17 08:48:19.000000000 -0600
+++ src/cdk-5.0-20060507/include/config.hin	2008-04-14 00:01:33.000000000 -0500
@@ -5,8 +5,6 @@
 #ifndef CDK_CONFIG_H
 #define CDK_CONFIG_H 1
 
-@DEFS@
-
 #if !defined(HAVE_LSTAT) && !defined(lstat)
 #define lstat(f,b) stat(f,b)
 #endif
